aptos-move/replay-benchmark/src/commands/initialize.rs

```diff
@@ -84,15 +93,17 @@ impl InitializeCommand {
         })?;
 
         // TODO:
-        //  Right now, only features can be overridden. In the future, we may want to support:
-        //      1. Framework code, e.g., to test performance of new natives or compiler,
-        //      2. Gas schedule, to track the costs of charging gas or tracking limits.
-        //      3. BlockExecutorConfigFromOnchain to experiment with different block cutting based
-        //         on gas limits.
+        //   1. Override gas schedule, to track the costs of charging gas or tracking limits.
+        //   2. BlockExecutorConfigFromOnchain to experiment with different block cutting based
+        //      on gas limits?.
+        //   3. Build options for package overrides.
```

Comment:
Ah, so this would allow pointing to just source. Perhaps mention that right now, the build option is only used for getting the bytecode version (and that this wont recompile the package with move-2).